include/WCSimDetectorConstruction.hh

```diff
@@ -89,6 +89,8 @@ public:
   G4double GetPMTSize1() {return WCPMTSize;}
 
   G4float GetPMTQE(G4float, G4int, G4float, G4float, G4float);
+  G4float GetPMTCE(G4float);
```

Comment:
Perhaps something is missing from this pull request?  I cannot see the implementation of this function anywhere in the commit.